SharedCode/utils.py

```diff
@@ -1,6 +1,7 @@
 """Functions shared by Azure Functions"""
 
 import os
+import html
```

Comment:
Is this needed?

Reply:
Yes, html.unescape used in get_english_welsh_item to unescape HTML entries in string such  <LOCNAME>Abingdon &amp;amp; Witney College (Common Leys Campus)</LOCNAME>